cryoet_data_portal_neuroglancer/state_generator.py

```diff
@@ -291,6 +293,17 @@ def combine_json_layers(
             "row": 2,
             "visible": False,
         },
+        "toolPalettes": {
```

Comment:
There are a few changes which would be helpful here! The first is to have it stack horizontally, which would be controlled by "verticalStacking". The second is to make this a query, because that way the user can't delete the controls.

Something like
```
"toolPalettes": {
    "Palette": {
      "side": "bottom",
      "row": 1,
      "size": 120,
      "query": "type:dimension",
      "verticalStacking": false
    }
  }
```
  
The `size` would depend on whether it is vertically or horizontally stacked to be honest, but let's try this size for now.